Update and rename Invite-Employees.md to Invite.Members.md (#28276)
Expensify/App: docs/articles/expensify-classic/getting-started/Invite-Members.md

---
title: Invite Members
description: Learn how add your employees to submit expenses in Expensify
---
# Overview

> To invite your employees in Expensify, simply add them as members to your Workspace.

Reviewer: ```suggestion
To invite your employees to Expensify, simply add them as members to your Workspace.
```

# How to Invite members to Expensify

## Inviting Members Manually

> Navigate to **Settings > Workspace > Group > *Workspace Name* > People** - then click Invite and enter the invitee's email address.

Reviewer: ```suggestion
Navigate to **Settings > Workspace > Group > *Workspace Name* > People** - then click **Invite** and enter the invitee's email address.
```

Reviewer: SO asks us to bold buttons

Indicate whether you want them to be an Employee, Admin, or Auditor on the Workspace.

> If you are utilizing the Advanced Approval feature, you can specify who members should submit their expense reports to and who an approver should send approved reports to for the next step in the approval process. If someone is the final approver, you can leave this field blank.

Reviewer 2: Can we have someone review this copy? I think there is some english copy mistake here. I asked Nikki as well to see if it was just me but she also thinks this is off (Grammarly also flags it). For example, I think it should say `you can specify to whom members should submit their expense reports`. The approver part sounds also off.

> If you are utilizing the Advanced Approval feature, you can specify who members should submit their expense reports to and who an approver should send approved reports to for the next step in the approval process. If someone is the final approver, you can leave this field blank.

Reviewer 3: "and who an approver should send approved reports to for the next step in the approval process" sounds off @NicMendonca, I think this still needs to be fixed?

Reviewer 2: I left the same comment previously [here](https://github.com/Expensify/App/pull/28276#pullrequestreview-1652089064), something is off with the copy. Should it say "you can specify to whom members should submit their expense reports"?

Author: @pecanoro but that's not the feature. In the Advanced approval feature you a) specify who members should submit their expense reports to  - this is already in the copy) and then b) who *that* approver, approvers and forwards their reports to for final approver

Reviewer 3: Hmm reading it over again it actually sounds fine to me, maybe I read it out of context and it sounded off, sorry!

Reviewer 2: I still believe there is something wrong with that sentence and it should be rephrased it since it's really hard to understand 😅

Reviewer: I agree that this sentence is long and difficult to read. I've added a suggestion on how to rephrase it below.

> If you are utilizing the Advanced Approval feature, you can specify who members should submit their expense reports to and who an approver should send approved reports to for the next step in the approval process. If someone is the final approver, you can leave this field blank.

Reviewer: ```suggestion
If you are utilizing the Advanced Approval feature and the invitee is an approver, you can use the "approves to" field to specify to whom they approve and forward reports to for additional approval.
```

Reviewer: @NicMendonca I assume this sentence is about the "Approves to" field on the invite modal, is that correct? If yes, I suggest updating the sentence to the above suggestion

## Inviting Members to a Workspace in Bulk 

Navigate to **Settings > Workspaces > Group > *Workspace Name* > People** - then click Invite and enter all of the email addresses separated by comma. Indicate whether you want them to be an Employee, Admin, or Auditor on the Workspace.

> If you are utilizing the Advanced Approval feature, you can specify who members should submit their expense reports to and who an approver should send approved reports to for the next step in the approval process. If someone is the final approver, you can leave this field blank.

Reviewer 2: Same as my previous comment.

> If you are utilizing the Advanced Approval feature, you can specify who each member should submit their expense reports to and who an approver should send approved reports to for the next step in the approval process. If someone is the final approver, you can leave this field blank.

Reviewer: ```suggestion

```

Another convenient method is to employ the spreadsheet bulk upload option for inviting members to a Workspace. This proves particularly helpful when initially configuring your system or when dealing with numerous member updates. Simply click the "Import from Spreadsheet" button and upload a file in formats such as .csv, .txt, .xls, or .xlsx to streamline the process.

After uploading the spreadsheet, we'll display a window where you can choose which columns to import and what they correspond to. These are the fields: 
- Email
- Role
- Custom Field 1
- Custom Field 2
- Submits To
- Approves To
- Approval Limit
- Over Limit Forward To
  

> Click the Import button and you're done. We will import the new members with the optional settings and update any already existing ones.

Reviewer: ```suggestion
Click the **Import** button and you're done. We will import the new members with the optional settings and update any already existing ones.
```

Reviewer: The SO says to bold buttons
